.github/workflows/python-package-conda.yml

```diff
@@ -3,7 +3,7 @@ name: Python Package using Conda
 on:
   #push:
   schedule:
-    - cron: "0 20 * * *"
```

Comment:
Hm, just curious: Why did you change this one to run monthly?

Reply:
Because it is failing every day and I wanted to disable it until I merge the new version I developed in a separate branch.

Reply:
Ah, makes sense.